tcp/server.go

```diff
@@ -88,12 +97,13 @@ func handleMsgProcessing(connTCP net.Conn, msg string, root string) {
 
 	err = qbin.Store(&doc)
 	if err != nil {
```

Comment:
If the error is `file contains 0x00 bytes` I'd like this to be the same string as from [upload_route.go:134](https://github.com/qbin-io/backend/blob/master/http/upload_route.go#L134)